mint-auditor/src/db/models/audited_mint.rs

```diff
@@ -102,7 +112,9 @@ impl AuditedMint {
             Err(Error::DepositAndMintMismatch(_)) => {
                 Counters::inc_num_mismatching_mints_and_deposits(conn)?;
             }
-
+            Err(Error::NotFound) => {
```

Comment:
The thought here was that when there is a deposit, and the mint is on a different token_id, this should result in a NotFound error, because it does not seem to be possible to distinguish by an error where the mint was onto the wrong tokenid, and when there is a coincidental nonce collision with a mint on the 2nd tokenId. After discussion offline, I agree that this should not be a counted error and should instead by monitored by the resulting pair of unaudited mint/deposit. Fixing.